lib/dal/src/func/runner.rs

```diff
@@ -487,9 +485,7 @@ impl FuncRunner {
         ) -> FuncRunnerResult<FuncRunner> {
             let func_id =
                 Func::find_intrinsic(ctx, super::intrinsics::IntrinsicFunc::Validation).await?;
-            let func = Func::get_by_id(ctx, func_id)
-                .await?
-                .ok_or(FuncRunnerError::FuncIntrinsicValidationMissing)?;
```

Comment:
We're not checking for this anywhere, and I think it might technically be unreachable since the find_intrinsic would have failed above if the func was not on the graph